### PR TITLE
Rename "ios" & "fxios" to "Safari" and "Firefox" for browser name

### PR DESCRIPTION
**Pull request by ryanalexander**
prettier fixed formatting also ^

### CODE REVIEW

**Comment by ryanalexander** [comment]
Fix for https://github.com/revoltchat/revite/issues/118

File: src/pages/login/forms/FormLogin.tsx [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+            callback={async data => {
                 const browser = detect();
                 let device_name;
                 if (browser) {
-                    const { name, os } = browser;
-                    if (window.isNative) {
-                        device_name = `Revolt Desktop on ${os}`;
-                    } else {
-                        device_name = `${name} on ${os}`;
-                    }
```
Comment (insertish) [inline src/pages/login/forms/FormLogin.tsx:26]: Why is the native detection being removed?

**Review by insertish — CHANGES_REQUESTED** [review]
This is how I'd imagine this looks:

```js
let { name, os } = browser;
if (window.isNative) {
	device_name = `Revolt Desktop on ${os}`;
} else {
	if (name === 'ios') name = 'safari';
	else if (name === 'fxios') name = 'firefox';

	device_name = `${name} on ${os}`;
}
```

File: src/pages/login/forms/FormLogin.tsx (hunk 1 above)
Comment (ryanalexander) [reply src/pages/login/forms/FormLogin.tsx:26]: That appears to be a typo

Comment (ryanalexander) [reply src/pages/login/forms/FormLogin.tsx:26]: I'll add a commit adding it back

File: src/pages/login/forms/FormLogin.tsx [hunk 2]
```diff
@@ -1,28 +1,30 @@
+import { Form } from "./Form";
 import { detect } from "detect-browser";
-import { useHistory } from "react-router-dom";
```
Comment (insertish) [inline src/pages/login/forms/FormLogin.tsx:2]: Fairly certain removing this will cause it to fail to compile, I'll fix locally.

**Review by insertish — APPROVED** [review]

**Pull request merged**